config/eduro-followme-vanjee.json

```diff
@@ -0,0 +1,83 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "driver": "osgar.followme:FollowMe",
+          "in": ["encoders", "scan", "emergency_stop", "pose2d", "buttons"],
+          "out": ["desired_speed"],
+          "init": {
+            "max_speed": 0.5,
+            "laser_pose2d": [0.13, 0.055, 0],
+            "scan_size": 1800,
+            "scan_fov_deg": -360
```

Comment:
I hope this is only temporary solution. As we discussed yesterday.

Reply:
this could be "final" solution for follow me, but it was the minimal change I was thinking of. I suppose there will be update once mounted on Pat-robot and also once lidar data will be needed in multiple modules.